lib/features/mailbox/presentation/mailbox_view_web.dart

```diff
@@ -467,4 +475,90 @@ class MailboxView extends GetWidget<MailboxController> with AppLoaderMixin, Popu
           ),
         ));
   }
+
+  Widget _buildAppGridDashboard(BuildContext context) {
+    return Column(
+      children: [
+        _buildGoToApplicationsCategory(context, MailboxCategories.appGrid),
+        AnimatedContainer(
+          padding: const EdgeInsets.only(top: 8),
+          duration: const Duration(milliseconds: 400),
+          child: controller.mailboxDashBoardController.appGridDashboardController.appDashboardExpandMode.value == ExpandMode.EXPAND
+            ? _buildAppGridInMailboxView(context)
+            : const Offstage())
+    ]);
+  }
+
+  Widget _buildGoToApplicationsCategory(BuildContext context, MailboxCategories categories) {
+    return Padding(
+      padding: EdgeInsets.only(
+        top: 8,
+        left: _responsiveUtils.isDesktop(context) ? 0 : 36,
+        right: _responsiveUtils.isDesktop(context) ? 0 : 28
+      ),
+      child: Row(
+        children: [
+          buildIconWeb(
+            splashRadius: 5,
+            iconPadding: EdgeInsets.zero,
+            minSize: 12,
+            iconSize: 28,
+            icon: SvgPicture.asset(
+              _imagePaths.icAppDashboard,
+              color: AppColor.primaryColor,
+              fit: BoxFit.fill
+            ),
+            tooltip: AppLocalizations.of(context).appGridTittle),
+          Expanded(
+            child: Padding(
+              padding: const EdgeInsets.only(left: 12),
+              child: Text(categories.getTitle(context),
+                maxLines: 1,
+                overflow: TextOverflow.ellipsis,
+                style: const TextStyle(
+                  fontSize: 16,
+                  color: AppColor.colorTextButton,
+                  fontWeight: FontWeight.w500
+                )
+              )
+            )
+          ),
+          buildIconWeb(
+            splashRadius: 5,
+            iconPadding: EdgeInsets.zero,
+            minSize: 12,
+            iconSize: 28,
+            icon: Obx(() => SvgPicture.asset(
+              controller.mailboxDashBoardController.appGridDashboardController.appDashboardExpandMode.value == ExpandMode.COLLAPSE
+                ? _imagePaths.icCollapseFolder
+                : _imagePaths.icExpandFolder,
+              color: AppColor.primaryColor,
+              fit: BoxFit.fill
+            )),
+            tooltip: AppLocalizations.of(context).appGridTittle,
+            onTap: () => controller.toggleMailboxCategories(categories)
+          ),
+        ]
+      )
+    );
+  }
+
+  Widget _buildAppGridInMailboxView(BuildContext context) {
+    return Obx(() {
+      final linagoraApps = controller.mailboxDashBoardController.appGridDashboardController.linagoraApplications.value;
+      if (linagoraApps != null && linagoraApps.apps.isNotEmpty) {
+        return ListView.separated(
```

Comment:
If you don't have lines between items, you shouldn't use `ListView.separated`. Please use `ListView.builder`